.asf.yaml

```diff
@@ -17,18 +17,6 @@
 
 notifications:
   pullrequests: commits@camel.apache.org
-  jira_options: link label comment
+  jira_options: link label=
 publish:
   whoami: asf-site
-github:
-  labels:
-    - camel
-    - integration
-    - java
-    - hacktoberfest
-  enabled_merge_buttons:
-    squash:  true
-    merge:   false
-    rebase:  true
-  features:
-    issues: true
```

Comment:
This has little to do with publishing the website, accidentally removed?

Reply:
yes.